.travis.yml

```diff
@@ -60,14 +60,17 @@ install:
     - conda install xtensor==0.20.8 -c conda-forge
     # Install CxxWrap
     # TODO: switch to standalone libcxxwrap-julia
-    - julia -E "using Pkg; Pkg.add(PackageSpec(name=\"CxxWrap\", rev=\"master\"))"
-    # Retrieve JlCxx_DIR from CxxWrap
-    - JlCxx_DIR=$(julia -E "using CxxWrap; joinpath(dirname(pathof(CxxWrap)), \"..\", \"deps\", \"usr\", \"lib\", \"cmake\", \"JlCxx\")")
-    - JlCxx_DIR=${JlCxx_DIR//\"/}
+    - git clone https://github.com/JuliaInterop/libcxxwrap-julia.git
+    - cd libcxxwrap-julia
+    - mkdir build; cd build
+    - cmake -D CMAKE_INSTALL_PREFIX=$HOME/miniconda ..
+    - make install
+    - cd ../../
+    - JLCXX_DIR=$CMAKE_INSTALL_PREFIX julia -E "using Pkg; Pkg.add(PackageSpec(name=\"CxxWrap\", rev=\"master\"))"
```

Comment:
Is it possible `$CMAKE_INSTALL_PREFIX` is empty?

Reply:
good catch! I this is fixed now - although the build is still failing.